darwin/Platform.c

```diff
@@ -22,6 +22,7 @@ in the source distribution for its full text.
 #include "zfs/ZfsArcMeter.h"
 #include "zfs/ZfsCompressedArcMeter.h"
 #include "DarwinProcessList.h"
+#include "CRT.h"
```

Comment:
Header includes are sorted alphabetical, files in parent directories preceeding those from subdirectories.

Please refer to the styleguide for the exact order (seems this file has still been missed when this was updated everywhere).